src/test/test-open-file.c

```diff
@@ -179,7 +179,7 @@ TEST(open_file_to_string) {
         r = open_file_to_string(of, &s);
 
         assert_se(r >= 0);
-        assert_se(streq(s, "/path\\:with\\:colon"));
+        ASSERT_STREQ(s, "/path\\:with\\:colon");
```

Comment:
Please drop this. Will conflict with #32181.

Reply:
dropped